src/model/trust.model.ts

```diff
@@ -69,7 +68,7 @@ interface TrustIndividual {
 
 interface TrustHistoricalBeneficialOwnerCommon {
   id?: string;
-  corporate_indicator: TrusteeType;
+  corporate_indicator: yesNoResponse;
```

Comment:
I have already done that (details in https://companieshouse.atlassian.net/browse/ROE-1919). I will update the PR to make this clear since I ended up doing my developer tests on that Jira.

But thanks Mark - this is a important point